README.md

```diff
@@ -18,6 +18,57 @@ From your [eZ Publish Platform](https://doc.ez.no/display/EZP/Installing+eZ+Publ
 php -d memory_limit=-1 composer.phar require --prefer-dist ezsystems/ez-matrix-bundle:dev-master
 ```
 
+### How to update content programmatically
+
+Here's an example on how to update the value of a matrix field for a content item. The field has two columns, and we are creating two rows of content:
+
+```
+$repository = $this->getContainer()->get( 'ezpublish.api.repository' );
+$contentService = $repository->getContentService();
+
+$repository->setCurrentUser( $repository->getUserService()->loadUser( 14 ) );
```

Comment:
See https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/Repository.php#L41

So you can now avoid having to eagerly load the user actually. But as this aims to support 5.x also, maybe just add a comment referring to how it can be done on newer versions.